bayesian/api/api_v2.py

```diff
@@ -184,10 +183,16 @@ def get(ecosystem, package, version):
 
             return response_template({'error': msg}, 202)
 
+        # Creating a set to be passed to the function unknown_package_flow
+        unknown_pkgs = set()
+        unknown_pkgs.add(normlize_packages(name=package, given_name=package,
```

Comment:
not necessary to use normalized_packages. There is a package model which you can use which has only name and version

Reply:
Actually I did it this way because in POST API this format is been used and then object has been used further, I wanted to use same function in GET & POST APIs as currently it has separate function to trigger flow for GET and POST.